src/L1/USDXBridgeAlt.sol

```diff
@@ -114,22 +115,21 @@ contract USDXBridgeAlt is Ownable, ReentrancyGuard {
             "USDX Bridge: Fee-on-transfer tokens not supported."
         );
         totalBridged[_stablecoin] += bridgeAmount;
-        /// Mint USDX
-        //l1USDX.mint(address(this), bridgeAmount);
+        // Mint USDX
+        l1USDX.mint(address(this), bridgeAmount);
```

Comment:
The reason this line was uncommented is to bypass the production restriction where only Hex Trust, with the granted minter role, can mint tokens (see https://github.com/hextrust/hex-stablecoin-usd/blob/main/contracts/ERC20WithRolesUpgradeable.sol#L96).

In the test setup, we use a forked mainnet and redeploy the contract, so the address changes with each test run. We cannot ask Hex Trust to grant permission to the new address every time.

Nevertheless, the test passes because we manually allocate tokens to the bridge contract using deal(address(usdx), address(usdxBridgeAlt), 100e18) (See https://github.com/Ozean-L2/Ozean-Contracts/blob/main/test/L1/USDXBridgeAltForkTest.t.sol#L21). So, even without the mint code, we can still simulate minting when running the test on a forked mainnet.